yt/chyt/controller/internal/jupyt/api.go

```diff
@@ -36,13 +64,20 @@ func GetEndpoint(a *api.API, ctx context.Context, alias string) (result GetEndpo
 		return
 	}
 
-	address, _, err := net.SplitHostPort(jobs.Jobs[0].Address)
+	job := jobs.Jobs[0]
+
+	address, _, err := net.SplitHostPort(job.Address)
```

Comment:
FYI: This works only if the job is running without any network isolation. Otherwise the job address differs from the node address.

Do not need to fix it now, because it hadn't worked before you changed the code either. But it makes sense to leave a comment for those who will read this code later.